arangod/RestHandler/RestCollectionHandler.cpp

```diff
@@ -449,19 +449,13 @@ RestStatus RestCollectionHandler::handleCommandPut() {
   TRI_ASSERT(coll);
 
   if (sub == "load") {
-    res = methods::Collections::load(_vocbase, coll.get());
-
-    if (res.ok()) {
-      bool cc = VelocyPackHelper::getBooleanValue(body, "count", true);
-      collectionRepresentation(
-          name, /*showProperties*/ false,
-          /*showFigures*/ FiguresType::None,
-          /*showCount*/ cc ? CountType::Standard : CountType::None);
-      return standardResponse();
-    } else {
-      generateError(res);
-      return RestStatus::DONE;
-    }
+    // "load" is a no-op starting with 3.9
```

Comment:
It is deprecated since 3.9 already, and a no-op since then: https://github.com/arangodb/docs/blob/main/3.9/release-notes-api-changes39.md#javascript-api